code/src/components/Form.js

```diff
@@ -0,0 +1,296 @@
+	const [breakupPerson, setBreakupPerson] = useState('');
+	const [shipDuration, setShipDuration] = useState('default');
+	const [breakupReason, setBreakupReason] = useState('');
+	const [traitsCheckboxes, setTraitsCheckboxes] = useState([])
+	const [getBackTogether, setGetBackTogether] = useState('')
+	const [dateFriend, setDateFriend] = useState('')
+	const [friendContactDetails, setFriendContactDetails] = useState('')
+	const [error, setError] = useState('')
+
+	const onSubmitting = (e) => {
+		e.preventDefault();
+	};
+
+	const handleUserNameChange = (event) => {
+		setUserName(event.target.value)
+	}
+
+	const handleBreakupReason = (event) => {
+		setBreakupReason(event.target.value)
+	}
+
+	const handleDateFriend = (event) => {
+		setDateFriend(event.target.value)
+	}
+	
+	const handleGetBackTogether = (event) => {
+		setGetBackTogether(event.target.value)
+	}
+
+	const handleFriendContactDetails = (event) => {
+		setFriendContactDetails(event.target.value)
+	}
+	
+
+
+	const nextPage = (value) => {
+		if (value === 'default' || !value || value === []) {
+			setError("Please enter an answer")
+		} else {
+			setError('')
+			setPage(page + 1);
+		}
+	}
+
+	const backPage = () => setPage(page - 1);
+
+	const breakUpper = () => {
+		if (breakupPerson === "I broke up with you.") {
+			return ["you", "me"]
+		}
+		return ["I", "you"]
+	}
+
+	const handleShipDuration = (event) => {
+		setShipDuration(event.target.value)
+	}
+
+	const handleTraitsCheckboxesChange = (trait) => {
+		if (traitsCheckboxes.includes(trait)) {
+			const filteredTraits = traitsCheckboxes.filter(item => item !== trait)
+			setTraitsCheckboxes(filteredTraits)
+		} else {
+			setTraitsCheckboxes([...traitsCheckboxes, trait])
+		}
+	}
+
+	const reconciliation = () => {
+		if (getBackTogether === "Absolutely!" || getBackTogether === "Maybe?") {
+			return true
+		}
+		return false
+	}
+
+	return (
+		<form onSubmit={onSubmitting}>
+			
+			{page === 0 && (
+				<div className='current-page'>
+					<Input
+						labelClassName={'text-label'}
+						label={<h2>{questions.nameQuestion}</h2>}
+						onChange={handleUserNameChange}
+						value={userName}
+						placeholder={'Your name'}
+					/>
+					<div>
+						<h3 className='error'>{error}</h3>
+						<Button
+							key={'button-next'}
+							onClick={() => nextPage(userName)}
+							text='Next'
+						/>
+					</div>
+				</div>
+			)}
+			{page === 1 && (
+				<div className='current-page select-container'>
+					<Select 
+						shipDuration = {shipDuration}
+						setShipDuration = {handleShipDuration}
+						durationQuestion = {questions.durationQuestion}
+						duration = {questions.duration}	
+					/>
+					<div>
+						<h3 className='error'>{error}</h3>
+						<Button
+							key={'button-back'}
+							onClick={backPage}
+							text='Back'
+						/>
+						<Button
+							key={'button-next'}
+							onClick={() => nextPage(shipDuration)}
+							text='Next'
+						/>

```

Comment:
This is such a clever way of re-using the buttons component!!! I'm definitely stealing your approach more for the future